hw3/01-characters.js

```diff
@@ -1,4 +1,75 @@
-// url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
 
-fetch(url);
+let list = document.querySelector("#list");
+
+const fetchData = (url) => {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const response = await fetch(url);
+      const data = await response.json();
+      resolve(data);
+    } catch (error) {
+      reject(error);
+    }
+  });
+};
+
+const createCharacterElements = (character) => {
+  let col = document.createElement("div");
+  col.classList.add("col-3", "p-0", "m-2");
```

Comment:
Great use of bootstrap column class, I would recommend adding classes to change the column width on smaller screen sizes for better responsive layout. Something like "col-sm-6" if you wanted to have two columns on mobile screen sizes.

Reply:
great suggestion, I guess I was in a rush writing this code, I didn't pay enough attention to those details.